src/client/src/orangehrmPerformancePlugin/components/EditTrackerLogModal.vue

```diff
@@ -0,0 +1,211 @@
+              'orangehrm-add-tracker-rating-button': true,
+              '--negative': true,
+              '--deselected': !!rating,
+            }"
+            @click="onClickNegative"
+          />
+        </div>
+      </oxd-form-row>
+      <oxd-form-row>
+        <oxd-input-field
+          v-model="trackerLog.comment"
+          type="textarea"
+          :label="$t('general.comment')"
+          :placeholder="$t('general.type_here')"
+          :rules="rules.comment"
+        />
+      </oxd-form-row>
+      <oxd-divider />
+      <oxd-form-actions class="orangehrm-form-action">
+        <required-text />
+        <oxd-button
+          display-type="ghost"
+          :label="$t('general.cancel')"
+          @click="onCancel"
+        />
+        <oxd-button
+          display-type="secondary"
+          :label="$t('general.save')"
+          type="submit"
+        />
+      </oxd-form-actions>
+    </oxd-form>
+  </oxd-dialog>
+</template>
+
+<script>
+import Dialog from '@ohrm/oxd/core/components/Dialog/Dialog';
+import {
+  required,
+  shouldNotExceedCharLength,
+} from '@/core/util/validation/rules';
+import {APIService} from '@/core/util/services/api.service';
+
+const trackerLogModel = {
+  log: '',
+  comment: '',
+};
+
+export default {
+  name: 'EditTrackerLogModal',
+  components: {
+    'oxd-dialog': Dialog,
+  },
+  props: {
+    trackerId: {
+      type: Number,
+      required: true,
+    },
+    trackerLogId: {
+      type: Number,
+      required: true,
+    },
+  },
+  emits: ['close'],
+  setup(props) {
+    const http = new APIService(
+      window.appGlobal.baseUrl,
+      `api/v2/performance/trackers/${props.trackerId}/logs`,
+    );
+    return {
+      http,
+    };
+  },
+  data() {
+    return {
+      isLoading: false,
+      rating: true,
+      trackerLog: {...trackerLogModel},
+      rules: {
+        log: [required, shouldNotExceedCharLength(150)],
+        comment: [required, shouldNotExceedCharLength(3000)],
+      },
+    };
+  },
+  beforeMount() {
+    this.isLoading = true;
+    this.http
+      .get(this.trackerLogId)
+      .then(response => {
+        const {data} = response.data;
+        this.trackerLog.log = data.log;
+        this.trackerLog.comment = data.comment;
+        this.rating = data.achievement === 1;
+      })
+      .finally(() => {
+        this.isLoading = false;
+      });
+  },
+  methods: {
+    onClickPositive() {
+      this.rating = true;
+    },
+    onClickNegative() {
+      this.rating = false;
+    },
+    onSave() {
+      this.isLoading = true;
+      this.http
+        .update(this.trackerLogId, {
+          log: this.trackerLog.log,
+          comment: this.trackerLog.comment,
+          achievement: this.rating ? 1 : 2,
+        })
+        .then(() => {
+          this.$toast.updateSuccess();
+          this.onCancel();
+        });
+    },
+    onCancel() {
+      this.$emit('close');
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
```

Comment:
similar styles? let's put in one external style sheets and link both

Reply:
done